Tests/JsonApiSerializerTests.cs

```diff
@@ -83,6 +83,22 @@ public void UsesQueryFieldsetExpressions()
             Assert.NotNull(result["data"][0]["attributes"]["location"]);
             Assert.Null(result["data"][0]["attributes"]["number-of-employees"]);
         }
+        
+        [Fact(DisplayName = "Uses query fieldset expressions also on relationships if specified")]
```

Comment:
I guess the "also" will lose its meaning pretty soon after this is merged :smile:

Reply:
I do not quite get what you mean here, I hope I resolved the issue.

Reply:
You did, it wasn't really an issue just a stylistic thing :smile: if you read the test title by itself, the "also" didn't make sense, it only did in the context of the other tests. It was a very minor thing anyway.

Reply:
Ahh I see. Didn't get it probably because english istn't my first language :)